packages/api-client/src/views/Request/RequestSidebarItem.vue

```diff
@@ -124,7 +124,7 @@ const item = computed<SidebarItem>(() => {
       entity: request,
       resourceTitle: 'Request',
       warning: 'This cannot be undone. You’re about to delete the request.',
-      children: request.examples,
+      children: request.examples.slice(1),
```

Comment:
wait doesnt slice return a shallow copy?

Reply:
you right, should we avoid it?

Reply:
most likely but i'd ping @geoffgscott or @amritk !